odoo/addons/base/models/ir_model.py

```diff
@@ -308,6 +308,10 @@ class IrModelFields(models.Model):
     relation = fields.Char(string='Object Relation',
                            help="For relationship fields, the technical name of the target model")
     relation_field = fields.Char(help="For one2many fields, the field on the target model that implement the opposite many2one relationship")
+    relation_m2o = fields.Many2one(
```

Comment:
Please use standard naming convention (suffix `_id`): `relation_field_id` looks clearer to me.